core/include/drivers/gpio.h

```diff
@@ -188,7 +191,7 @@ typedef struct gpio *(*gpio_dt_get_func)(struct dt_driver_phandle_args *a,
  * @fdt: Device tree to work on
  * @nodeoffset: Node offset of the GPIO controller
  * @get_dt_gpio: Callback to match the GPIO controller with a struct gpio
- * @gpio_chip: data which will be passed to the get_dt_gpio callback
+ * @data: Opaque reference which will be passed to the get_dt_gpio callback
```

Comment:
This change belongs to commit "drivers: gpio: add device-tree based gpio controller framework".